### ISSUE TITLE
Media: "Write Post" action does nothing on media upload notice in Media section

### ISSUE THREAD

**Issue by rachelmcr** [wordpress-mobile/WordPress-iOS#11347]
### Expected behavior

I expect to upload media in the Media section, see a "Media uploaded" notice when the media upload is successful, and tap "Write Post" to open the editor with the media added.

### Actual behavior

The media uploads and I see the notice, but the "Write Post" action does nothing. (This happens regardless of whether I am using Aztec or the block editor in the app.)

### Steps to reproduce the behavior

1. Open a site's Media section in the app.
1. Add one or more media items using the + button.
1. When the uploads finish and you see a "Media uploaded" notice, tap on "Write Post" on the notice.

![2019-03-27 17 15 50](https://user-images.githubusercontent.com/8658164/55097651-2bc24c80-50b4-11e9-86ce-79503215ff01.png)

##### Tested on [iPhone XS and iPad Mini], iOS [12.1.4], WPiOS [12.1.0.20190325]
Also confirmed that this is working for me in the 12.0 app store version, so the issue is new to the 12.1 beta.

cc @SergioEstevao

**Comment by SergioEstevao** [comment]
@rachelmcr just tested both editor on the release/12.1 branch and it's working fine for me... can you try to rebuild?

**Comment by rachelmcr** [comment]
@SergioEstevao It works for me when I test on the release/12.1 branch in a simulator, but not with the WP Internal beta release on real devices.

**Comment by rachelmcr** [comment]
While testing today, I confirmed the issue was still happening. However, I then force closed the app and reopened it, and the "Write Post" action started working. Could this be an issue that only appears when updating from the previous version, and restarting the app resolves it?

**Comment by SergioEstevao** [comment]
It shouldn't the code has no condition that should force that.

**Comment by designsimply** [comment]
I tested with WP Internal 12.1.0.20190325, both before and after upgrading to WP Internal 12.1.0.20190328 via HockeyApp, on iPhone 6S iOS 12.2 and confirmed "Write Post" after uploading media is working for me. I tested the "Free Photo Library", "Other Apps", and "Choose from My Device" options from 4 different types of sites: private WordPress.com Free, WordPress.com Free, Jetpack 7.1.1 WP 5.1.1, and Self-hosted WP 5.1.1 and it worked every time! ()

HOWEVER! I re-tried to make sure I had tested every which way to add media and the "Take Photo or Video" option failed every time for every type of site on WP Internal 12.1.0.20190325 on iPhone 6S iOS 12.2. ([2m36s](https://cloudup.com/c8a-M1_g7Gf))

My testing steps:

1. Go to My Sites > Media > +.
1. Select "Take Photo or Video".
1. Take a photo.
1. Select "Use Photo".
1. Wait for the photo to upload and for the "Write Post" prompt to appear and try to tap on "Write Post".

Result: the "Write Post" link is unresponsive after choosing the "Take Photo or Video" option when uploading images in the Media section. ([1m44s](https://cloudup.com/c4i5LO9vaWZ))

<img src="https://user-images.githubusercontent.com/1119271/55197816-4f9b9600-5179-11e9-9f70-398e69488910.png" width="270" alt="take-photo-or-video">   <img src="https://user-images.githubusercontent.com/1119271/55197814-4f9b9600-5179-11e9-824f-d33be79c8aeb.png" width="270" alt="write-post">
<sup>Tested with WP Internal 12.1.0.20190328 on iPhone 6S iOS 12.2.</sup>
